static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -159,6 +159,8 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       this.searchThisArea();
     });
 
+    this._pageWrapperEl.classList.add('YxtPage-wrapper--listShown');
```

Comment:
Instead of adding it through JavaScript on initial page load. Do we want to add this CSS class to the page directly on vertical-full-page-map pages?

Reply:
That would be ideal, however YxtPage-wrapper is defined inside layotus/html.hbs, and this seems like a better solution than conditionally adding YxtPage-wrapper--listShown to layouts/html.hbs

Reply:
My worry is that this can cause a flash of CSS on page load, where you might see the map and then it hides away after a second. Is your worry about adding some `pageType` parameter to the >layouts/html partial?

Reply:
The `pageType` parameter idea actually sounds like a good solution. I was more concerned about putting the 'VerticalFullPageMap' string somewhere in layouts/html.hbs, but we could avoid that with a param. I'll try that.

I tested it though, and no page flashing occurs because the CSS class is added to the page before the map component is added

Reply:
Updated so that YxtPage-wrapper-mobileListView is default, however if the page loads on desktop, we remove the CSS class. This causes no visible CSS flashing.